web/src/main/java/com/garret/movies/MovieApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.garret.movies;
+
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.PropertySource;
+
+@SpringBootApplication
+@EnableBatchProcessing
+@PropertySource(value = {"application.properties", "omdb.properties", "dao.properties"})
```

Comment:
@Garret-vin Application properties by default already in context, about others please add property source on top of beans which holds this config